README.md

```
@@ -218,161 +218,161 @@ By following these guidelines, you can extend this project to handle different t
+| orig_projected_additional_accrued_interest|         |
+| hardship_payoff_balance_amount|                     |
+| hardship_last_payment_amount |                      |
+| disbursement_method          |                      |
+| debt_settlement_flag         |                      |
+| debt_settlement_flag_date    |                      |
+| settlement_status            |                      |
+| settlement_date              |                      |
+| settlement_amount            |                      |
+| settlement_percentage        |                      |
+| settlement_term              |                      |
+A sample file has been generated using ChatGPT.
```

markdownlint-cli2 (0.17.2)

Table pipe style
Expected: leading_and_trailing; Actual: no_leading_or_trailing; Missing leading pipe

(MD055, table-pipe-style)